qa/L0_function_stack/test_pyprof_func_stack.py

```diff
@@ -49,7 +49,7 @@ def remove_test_class_hierarchy(x):
             # Find the LAST instance of run in the split
             #
             for i, n in enumerate(fn_split):
-                if (n == "TestPyProfFuncStack::run"):
+                if (n == "TestPyProfFuncStack::_callTestMethod"):
```

Comment:
Comment on line 43 needs to be updated or generalized

Reply:
done